it/stubs/PenaltiesStub.scala

```diff
@@ -18,84 +18,21 @@ package stubs
+
 import java.time.{LocalDate, LocalDateTime}
 
 object PenaltiesStub {
   val vrn: String = "HMRC-MTD-VAT~VRN~123456789"
-  val getLspDataUrl: String = s"/penalties/etmp/penalties/$vrn\\?newApiModel=false&newFinancialApiModel=false"
-  val getLspDataUrlAgent: String = s"/penalties/etmp/penalties/$vrn\\?arn=123456789&newApiModel=false&newFinancialApiModel=false"
   val sampleDate1: LocalDateTime = LocalDateTime.of(2021, 1, 1, 1, 1, 1)
 
-  val getPenaltyDetailsUrl1811ModelEnabled: String = s"/penalties/etmp/penalties/$vrn\\?newApiModel=true&newFinancialApiModel=true"
-  val getPenaltyDetailsUrlAgent1811ModelEnabled: String = s"/penalties/etmp/penalties/$vrn\\?arn=123456789&newApiModel=true&newFinancialApiModel=true"
-  val getPenaltyDetailsUrl1811ModelDisabled: String = s"/penalties/etmp/penalties/$vrn\\?newApiModel=true&newFinancialApiModel=false"
-  val getPenaltyDetailsUrlAgent1811ModelDisabled: String = s"/penalties/etmp/penalties/$vrn\\?arn=123456789&newApiModel=true&newFinancialApiModel=false"
-
-  val sampleLspData: ETMPPayload = ETMPPayload(
-    pointsTotal = 0,
-    lateSubmissions = 0,
-    adjustmentPointsTotal = 0,
-    fixedPenaltyAmount = 0.0,
-    penaltyAmountsTotal = 0.0,
-    penaltyPointsThreshold = 4,
-    penaltyPoints = Seq.empty[PenaltyPoint],
-    latePaymentPenalties = Option(Seq.empty[LatePaymentPenalty])
-  )
-
-  val sampleLspDataWithMultiplePenaltyPeriod: ETMPPayload = ETMPPayload(
-    pointsTotal = 0,
-    lateSubmissions = 0,
-    adjustmentPointsTotal = 0,
-    fixedPenaltyAmount = 0.0,
-    penaltyAmountsTotal = 0.0,
-    penaltyPointsThreshold = 4,
-    penaltyPoints = Seq(
-      PenaltyPoint(
-        `type` = PenaltyTypeEnum.Point,
-        id = "1234",
-        number = "1",
-        appealStatus = None,
-        dateCreated = sampleDate1,
-        dateExpired = Some(sampleDate1),
-        status = PointStatusEnum.Active,
-        reason = None,
-        period = Some(
-          Seq(PenaltyPeriod(
-            startDate = sampleDate1,
-            endDate = sampleDate1.plusDays(15),
-            submission = Submission(
-              dueDate = sampleDate1.plusMonths(4).plusDays(7),
-              submittedDate = Some(sampleDate1.plusMonths(4).plusDays(12)),
-              status = SubmissionStatusEnum.Submitted
-            )
-          ),
-            PenaltyPeriod(
-              startDate = sampleDate1.plusDays(16),
-              endDate = sampleDate1.plusDays(31),
-              submission = Submission(
-                dueDate = sampleDate1.plusMonths(4).plusDays(23),
-                submittedDate = Some(sampleDate1.plusMonths(4).plusDays(25)),
-                status = SubmissionStatusEnum.Submitted
-              )
-            )
-          )
-        ),
-        communications = Seq.empty,
-        financial = None
-      )
-    ),
-    latePaymentPenalties = Option(Seq.empty[LatePaymentPenalty])
-  )
+  val getPenaltyDetailsUrlVatTrader: String = s"/penalties/etmp/penalties/$vrn"
```

Comment:
Do we want to rename this to: 
```suggestion
  val getPenaltyDetailsUrlVATTrader: String = s"/penalties/etmp/penalties/$vrn"
```